.github/workflows/example-get-started-deploy.yaml

```diff
@@ -0,0 +1,61 @@
+name: example-get-started deploy
+on: 
+  push:
+    paths:
+      - example-get-started/**
+    branches:
+      - master
+  workflow_dispatch:
+permissions:
+  contents: write
+  id-token: write
+  pull-requests: write
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    environment: aws
+    steps:
+    - name: Checkout repo
+      uses: actions/checkout@v3
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.11'
+        cache: 'pip'
+    - uses: aws-actions/configure-aws-credentials@v2
+      with:
+        aws-region: us-east-2
+        role-to-assume: ${{ vars.AWS_SANDBOX_ROLE }}
+        role-duration-seconds: 43200
+    - uses: iterative/setup-dvc@v1
+    - name: Generate repo
+      run: |
+        pip install virtualenv
+        cd example-get-started
+        ./generate.sh true
+    - name: Deploy repo
+      env:
+        REPO_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        # add remote
+        cd build/example-get-started
+        git remote add origin git@github.com:iterative/example-get-started.git
+        # close open PRs
+        gh pr close try-large-dataset
+        gh pr close tune-hyperparams
+        # force push branches
+        git push --force origin main
+        git push --force origin try-large-dataset
+        git push --force origin tune-hyperparams
+        # we push git tags one by one for Studio to receive webhooks:
```

Comment:
🤔 It's only relevant if we stop using some old tag, right? Not sure how to do it since pulling the tags locally will likely fail since they conflict with the new tags.

Reply:
I think we could do it like `git ls-remote --tags origin | awk '{print $2}' | xargs -n 1 git push --delete origin || true`. Do you think it's worth it?

Reply:
yep, I think so, why not?

Reply:
Okay, added it for all refs (not just tags) and dropped the other calls so it deletes old tags, branches, and exp refs with one line.